tests/integration/analytics/test_collector.py

```diff
@@ -0,0 +1,203 @@
+#  Copyright 2025 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import json
+from datetime import datetime, timezone
+from unittest.mock import ANY, MagicMock, patch
+
+import pytest
+from django.db import connection
+
+from aap_eda.analytics.collector import AnalyticsCollector, gather
+
+
+@pytest.fixture
```

Comment:
should we make it available through conftest.py?

Reply:
> should we make it available through conftest.py?

The mock_setting defined locally is more specific and clean for tests on the collector.